examples/qml_features/rust/src/empty_bridge.rs

```diff
@@ -0,0 +1,3 @@
+#[cxx_qt::bridge]
```

Comment:
this is missing a SPDX header, copy one from elsewhere and update with your name etc :-)